visa_mock/base/base_mocker.py

```diff
@@ -199,7 +199,7 @@ def send(self, scpi_string: str) -> Any:
         handler = None
 
         for regex_pattern in self.__scpi_dict__:
-            search_result = re.search(regex_pattern, scpi_string)
+            search_result = re.match(make(regex_pattern), scpi_string, re.IGNORECASE)
```

Comment:
There is still a problem here... you are calling "make" every time you are looking for a match. Why not make sure that `__scpi_dict__` is populated with the correct regular expressions in the first place? 

Have a look at line 177 of "base_mocker.py"

Reply:
The reason I didn't touch the original `__scpi_dict__` is that I want to keep the user input when calling the `set_call_delay`. But I decided to make the changes to that function too, see line 168.